_studio/shared/umc/io/umc_va/src/umc_va_linux.cpp

```diff
@@ -176,15 +176,18 @@ VAProfile get_next_va_profile(uint32_t umc_codec, uint32_t profile)
     case UMC::VA_H265:
         if (profile < UMC_ARRAY_SIZE(g_H265Profiles)) va_profile = g_H265Profiles[profile];
         break;
+#if (MFX_VERSION >= 1027)
```

Comment:
Imho, these direct version checks in the code look ugly. The problem is that sometimes it is difficult to say why code is covered by this. I would prefer (and I think this was the policy in a way) to have macro defs like MFX_ENABLE_H264_ROUNDING_OFFSET.

Reply:
Talked to @dmitry-gurulev regarding this. We introduce version checks here according to the changes in the API.